Packs/FeedCyjax/IndicatorFields/indicatorfield-Techniques_Tactics_Procedures.json

```
@@ -0,0 +1,26 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "techniquestacticsprocedures",
```

Please change it to:
```suggestion
    "cliName": "cyjaxtechniquestacticsprocedures",
```